lib/routes.json

```diff
@@ -1885,6 +1885,7 @@
                 "$number": null,
                 "body": {
                     "type": "Array",
+                    "sendValueAsBody": true,
```

Comment:
hmm does this really work ? Your condition expects a format json, but this body is a Array.